optimal_control_python/up_and_down_bow.py

```diff
@@ -125,28 +124,17 @@ def prepare_nlp(biorbd_model_path="../models/BrasViolon.bioMod", show_online_opt
 
     # Constraints
     constraints = (
-        (
-            Constraint.Type.MARKERS_TO_PAIR,
-            Constraint.Instant.START,
-            (ViolinString.marker_frog, violon_string.bridge_marker),
-        ),
-        (
-            Constraint.Type.MARKERS_TO_PAIR,
-            Constraint.Instant.MID,
-            (ViolinString.marker_tip, violon_string.bridge_marker),
-        ),
-        (
-            Constraint.Type.MARKERS_TO_PAIR,
-            Constraint.Instant.END,
-            (ViolinString.marker_frog, violon_string.bridge_marker),
-        ),
+        # (Constraint.Type.MARKERS_TO_PAIR, Constraint.Instant.START, (Bow.frog_marker, violon_string.bridge_marker),),
+        # (Constraint.Type.MARKERS_TO_PAIR, Constraint.Instant.MID, (Bow.tip_marker, violon_string.bridge_marker),),
+        # (Constraint.Type.MARKERS_TO_PAIR, Constraint.Instant.END, (Bow.frog_marker, violon_string.bridge_marker),),
+        (Constraint.Type.ALIGN_WITH_CUSTOM_RT, Constraint.Instant.ALL, (Bow.segment_idx, violon_string.rt_on_string),),
```

Comment:
Could you add in BiorbdOptim the fact that we can send a dict for the 3rd argument (such as Objective)